test/specs/modules/Modal/Modal-test.js

```diff
@@ -12,36 +12,21 @@ describe('Modal', () => {
   common.rendersChildren(Modal)
   common.hasSubComponents(Modal, [ModalHeader, ModalContent, ModalFooter])
 
-  describe('initial state', () => {
-    it('isShown is false', () => {
-      shallow(<Modal />)
-        .should.have.state('isShown', false)
-    })
-  })
-
-  describe('showModal()', () => {
-    it('sets isShown true', () => {
-      const modal = shallow(<Modal />)
-
-      modal
-        .instance()
-        .showModal()
-
-      modal
-        .should.have.state('isShown', true)
-    })
-  })
-  describe('hideModal()', () => {
-    it('sets isShown false', () => {
-      const modal = shallow(<Modal />)
-
-      modal
-        .instance()
-        .hideModal()
-
-      modal
-        .should.have.state('isShown', false)
-    })
+  common.propKeyOnlyToClassName(Modal, 'active')
+  common.propKeyOnlyToClassName(Modal, 'basic')
+  common.propKeyOnlyToClassName(Modal, 'long')
+  common.propValueOnlyToClassName(Modal, 'size')
+
+  it('renders a <div /> element', () => {
+    shallow(
+      <Modal>
+        <Modal.Header />
+        <Modal.Content>
+          <div />
+        </Modal.Content>
+        <Modal.Footer />
+      </Modal>
+    ).should.have.tagName('div')
```

Comment:
Since the contents of this tests are not used in the assertion, let's only render what is needed to make the assertion:

```jsx
shallow(<Modal />)
  .should.have.tagName('div')
```

The tests for the other components will assert that their className is correct and that they render their children.